conf/muchos.props.example

```diff
@@ -108,6 +108,29 @@ profile=perf-small
 # Below are different performance profiles that can be selected.  Each profile
 # has the same properties with different values.
 
+[azure]
```

Comment:
I think it would be nice to add some comments to the props in this section.  I took a stab at this and found I had some questions in the process.  My questions are intermingled.

```
[azure]
# The name of an Azure resource group to create.  All resources are created
# within this group. Question, what happens when it already exists?  
resource_group = accumulo-rg
# The name of an Azure virtual network that will be created. Question is it
# ok if this already exist in the RG?
vnet = vnet1
# The CIDR prefix used to create the virtual network. 
vnet_cidr = "10.0.0.0/8"
# A single subnet is created within the virtual network and given the 
# following name.
subnet = subnet1
# The CIDR prefix used for the single subnet within the virtual network.
subnet_cidr = "10.1.0.0/16"
# A virtual machine scale set with this many VMs will be created.  
# Question, what is the minimum allowed?  Seems like at lest three are 
# needed looking at create_vmss.yml 
numnodes = 8
# The size of each virtual machine.  See  the following like for other sizes :
# https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes-general
# Question, should this prop be vm_size instead of vm_sku?
vm_sku = Standard_D8s_v3
# Each VM will have the following type of managed disk
managed_disk_type = Standard_LRS
# Number of managed disk to create on each VM.
numdisks = 3
# The size of each managed disk in a VM.
disk_size_gb = 128
# All managed disk will be mounted at this location in each VM
mount_root = /var/data
# What is this?
metrics_drive_root = var-data
```

Reply:
👍 on the comments, will push up an update addressing some of this. To answer a few of the questions here:
- We'll use the existing resource_group and/or vnet if one exists with the same name. If it doesn't exist, it will be created. That's intentional since quite often folks will be given access restricted to a resource_group in Azure.
- Yes, the minimum number of nodes supported currently is 3. Will double-check and put in a comment about this. 
- Size / SKU are used interchangeably in Azure parlance, so the vm_sku property name should be okay.

Reply:
Done